### ISSUE TITLE
Culling of stratholme ascended mode holy water

### ISSUE THREAD

**Issue by danniellos94** [gamefreedomgit/Frostmourne#2029]
**Description:**
Holy water have the '?' icon and can't be bind on the skill bar.

**How to reproduce:**
Do culling of stratholme ascended.

**How it should work:**
You should be able to see the icon of holy water and be able to bind it to skill bar.

**Comment by Manpower99** [comment]
In addition to this, the item should be similar to the quest item chromie gives you, and removed upon leaving culling of stratholme.

**Comment by Aeradel** [comment]
Not able to replicate this. I was able to both see the item and place it on my hotbar. Are you using the 3.5v Whitemane client?

**Comment by UltraNix** [comment]
@danniellos94 Do you have the newest patch?

**Comment by danniellos94** [comment]
Yes although it was on hd client but friend had normal client and same issue so not sure.

**Comment by Aeradel** [comment]
> Yes although it was on hd client but friend had normal client and same issue so not sure, I'm also able to use the item outside of the dungeon.
> 
> https://ibb.co/yhc61PJ

You need the most recent Whitemane client to properly view these items. 

You can download this client at Whitemane.org/en

Your friend very likely has an older client with the patch, or not the Whitemane client.

**Comment by danniellos94** [comment]
Possible, still at least fix the water being outside usable of the dungeon the best way would be disappear after leaving dungeon I guess.

**Comment by Manpower99** [comment]
![image](https://github.com/user-attachments/assets/54d5764c-d738-45f5-a345-24d1f2a24959)
Should be 1.5s Cooldown.
in addition to this it has the "Culling of Stratholme" tag which should be sufficient proof that similar to this item 
![image](https://github.com/user-attachments/assets/22e56d40-c05c-40ca-8269-fd1de44a96a1)
It should be deleted from ur inventory if your Body/Corpse isn't inside the dungeon itself.

**Comment by Mynameistakenalready** [comment]
Previous tickets in game lead to nothing more than going here. Following intructions had from GMs(on normal client):
Latest patch -done
Cleared cache - done
Go to github and wrote about this bug - right now DONE.
3 months still problem.

**Comment by Aeradel** [comment]
> Previous tickets in game lead to nothing more than going here. Following intructions had from GMs(on normal client):
> 
> Latest patch -done
> 
> Cleared cache - done
> 
> Go to github and wrote about this bug - right now DONE.
> 
> 3 months still problem.

This seems isolated. If you're using the Whitemane client, I would ensure you're using reliable addons. Addons can interfere heavily. Even just disabling all of them to test if it's still a relevant issue helps.
